templates/base.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
This entire file serves as a **base template** for the rest of the pages. This file uses _bootstrap_ to create our navbar, which can then be inherited by the other pages.